README.md

```diff
@@ -37,7 +37,7 @@ eikooc
 
 # Course description
 
-## Lectures' outline
+## Syllabus 
 
 Please note, this outline is an approximation for the course schedule. At e-conomic we take a lean approach to software development, and we will take the same approach with the course; we can make adjusments to the lectures as the course progresses.
```

Comment:
How about deleting "At e-conomic we take a lean approach to software development, and we will take the same approach with the course;" Seems a bit irrelevant here. The point is that it is an approximation subject to change.